config/index.js

```diff
@@ -17,6 +17,9 @@ var conf = convict({
     format: [ 'dev', 'test', 'stage', 'prod' ],
     env: 'NODE_ENV'
   },
+  geodb: {
+    dbPath: path.resolve(__dirname, '../node_modules/fxa-geodb/db/cities-db.mmdb')
```

Comment:
See other config values. for example:
```
    idle: {
      doc: 'Idle timeout for memcached connections (milliseconds)',
      format: Number,
      default: 30000,
      env: 'MEMCACHE_METRICS_CONTEXT_IDLE'
    },
```

so in your case it would be `dbPath`
`format: String`, `doc: 'Path to the maxmind database file` (or something like that). default would be the path you provided (the node_module path)